spec/features/creating_peeps_spec.rb

```diff
@@ -0,0 +1,30 @@
+feature 'Creating Peeps' do
+
+  let(:user) do
+    User.create(name: 'test',
+                username: 'test',
+                email: 'test@test.com',
+                password: '12345',
+                password_confirmation: '12345')
```

Comment:
you are using Factory Girl so don't need to add the user fields. They are in the factory already. This just overwrites them